libs/vkd3d/utils.c

```diff
@@ -153,6 +153,239 @@ static const struct vkd3d_format vkd3d_depth_stencil_formats[] =
 #undef SINT
 #undef UINT
 
+static const struct vkd3d_format_compatibility_info_2
+{
+    DXGI_FORMAT image_format;
+    DXGI_FORMAT view_formats[VKD3D_MAX_COMPATIBLE_FORMAT_COUNT];
+    DXGI_FORMAT uint_format;
```

Comment:
Reading later commits, I understood what this was used for, but would be good to add a comment saying it's to handle ClearUAVUint.